web-app-router/components/Input/index.tsx

```diff
@@ -1,7 +1,99 @@
-export const Input = () => {
+"use client";
+import clsx from "clsx";
+import { FloatingLabel } from "flowbite-react";
+import { ComponentProps, memo } from "react";
+import { Icon } from "../Icon";
+
+type InputProps = ComponentProps<"input"> & {
+  type?: ComponentProps<"input">["type"];
+  disabled?: boolean;
+  helperText?: string;
+  label: string;
+  className?: string;
+  placeholder?: string;
+  required?: boolean;
```

Comment:
@andy-t-wang we don't need this prop in case we will use react-hook-form + yup. This will be defined by yup schema.

Reply:
I actually think it's more comfortable to use required directly since hook form doesn't pass in the yup required field. You need to pass it in register but even that was finnicky to me. Lmk what you think I made a change in my new update

Reply:
I don't see why we need to duplicate validation rules on the inputs directly while we already have them in the schema